client/src/test/java/io/pravega/client/segment/impl/SegmentOutputStreamTest.java

```diff
@@ -248,7 +248,6 @@ public void testConnectWithMultipleFailures() throws Exception {
         ClientConnection connection = mock(ClientConnection.class);
         InOrder verify = inOrder(connection);
         cf.provideConnection(uri, connection);
-        @Cleanup
```

Comment:
Why?

Reply:
Valid point. I have ensured close is invoked in the finally block for all the tests.